codes/go/pkg/print_utils.go

```diff
@@ -96,3 +96,10 @@ func showTrunk(t *trunk) {
 	showTrunk(t.prev)
 	fmt.Print(t.str)
 }
+
+// PrintHashMap Print a hash map
+func PrintMap(m map[int]string) {
```

Comment:
Please implement a function with Generics, ie., we should not limit the type of key and value.

Reply:
I added PrintMap by reference to PrintSlice function. If Generics are required, does PrintSlice function also need to be supported?

Reply:
Sorry that I missed this comment. Yes, I think it should also support generics. Could you help update it? Thanks!

Reply:
No problem! @krahets